lib/context-menu-and-spellcheck.js

```diff
@@ -95,6 +95,16 @@ function setupContextMenuAndSpellCheck (config, { navigate, get }) {
           }
         })
         menu.append(copyEmbed)
+        const openImageInBrowser = new MenuItem({
+          label: 'Open Image with Browser',
```

Comment:
Is there a precedent in Patchwork regarding whether to capitalize the `w` in with in a context menu item? From a quick glance at context menus on various elements, I see no other uncapitalized words. Some menu items on my Mac feature a capitalized `With`:
<img width="259" alt="Check Grammar With Spelling" src="https://user-images.githubusercontent.com/3607279/63381571-f53ce500-c366-11e9-90c8-c8a0b654d853.png">
Thoughts?

Reply:
Thanks for the review! :eyes: If you highlight text and right-click it does "Search with DuckDuckGo", so I followed that precedent, but I'm happy to switch both to "With" if you'd prefer.

I have absolutely no preferences here, so if you're leaning one way or another please feel free to change it. :bowing_man:

Reply:
Looking to Firefox for inspiration I've found `Search Google for "string..."` which, on consideration, sounds terrible because it's implying that what you're searching _is Google_, instead of that you're _using_ Google to search.

_(unless "Google" is now an adverb as well as a verb as well as the original proper noun. 🙃)_

I prefer capital-W-With, based on:
* capitalization patterns I'm seeing in the rest of Patchwork (only shorter words like `for` and `in` are uncapitalized)
* macOS 10.14.6 menu formatting (`Open With...` in finder, afformentioned `check Grammar With Spelling`)

Added commit!

Reply:
Looks really great to me, thanks for taking care of that! It looks great. :+1: